### DISCUSSION TITLE
trouble-shooting/web-worker-timer-refactoring/

### DISCUSSION THREAD

**Discussion by giscus** [rbgksqkr/rbgksqkr.github.io#1] [Announcements]
# trouble-shooting/web-worker-timer-refactoring/

문제 상황을 이해하며, Web Worker의 필요성을 이해할 수 있다.

https://rbgksqkr.github.io/trouble-shooting/web-worker-timer-refactoring/

**Comment by rbgksqkr** [comment]
댓글 테스트~!
